### DISCUSSION TITLE
MOV converter not found

### DISCUSSION THREAD

**Discussion by livvielicious** [ReagentX/imessage-exporter#398] [Q&A] [answered]
I'm getting this message when I try to run an export as  -f full or -f clone: "No MOV converter found, video attachments will not be converted!" Is there a converter I can download to fix that?

**Comment by ReagentX** [comment, marked as answer]
See the details [here](https://github.com/ReagentX/imessage-exporter/blob/develop/imessage-exporter/README.md#cross-platform-attachment-conversion). Note that with `clone`, attachments will be copied but not converted.

**Comment by livvielicious** [comment]
I'm using a Mac and chrome, so I'm not sure those things apply? I was hoping to have them converted to JPEG and MOV files, but even in "clone" im getting that MOV error. (Sorry if this is a dumb question, I am NOT a computers person, but I really need to preserve this chat history while freeing up space on my devices, so I'm trying my best!)

**Reply by ReagentX** [reply]
From the linked docs (emphasis added)

> [ffmpeg](https://ffmpeg.org/) is required to make exported audio more compatible on non-macOS platforms and **exported video more compatible on all platforms**.

If you are using `clone`, it doesn't matter, because the converter won't get used; `imessage-exporter` still emits the warning so that you know the feature is not available.

**Comment by ej-mediacenter** [comment]
I'm getting the same message when using copy method 'full' and I'm using macOS. Any advice?

**Reply by ReagentX** [reply]
Did you follow the instructions [here](https://github.com/ReagentX/imessage-exporter/blob/develop/imessage-exporter/README.md#cross-platform-attachment-conversion)?

**Reply by ej-mediacenter** [reply]
To the best of my knowledge, yes, but I'm a novice when it comes to dealing with code

<img width="737" alt="image" src="https://github.com/user-attachments/assets/f496a377-3427-44dd-a827-31e4c3954896" />

I've used [this video](https://www.youtube.com/watch?v=tyeCdSX2IFE) as a guide in addition to the instructions because I'm unfamiliar with coding

**Reply by ReagentX** [reply]
How did you install `ffmpeg`?
